### PR TITLE
Added product listing page

### PR DESCRIPTION
**Pull request by syed-shaida-hussain**
Please review.

Screenshot (48)

### CODE REVIEW

File: Product-listing-page/products.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    min-width: 10rem;
+    background: black;
+    border: none;
+    border-radius: 3px;
+}
+
+input[type="range"]::-webkit-slider-thumb {
+    -webkit-appearance: none;
+    border: none;
+    height: 16px;
```
Comment (tushargoyal812) [inline Product-listing-page/products.css]: use rem instead of px as vikram told.

**Review by tushargoyal812 — COMMENTED** [review]
design is good,do minor changes,looks good to merge

File: Product-listing-page/products.css (hunk 1 above)
Comment (syed-shaida-hussain) [reply Product-listing-page/products.css]: Will do it .Thank you

File: Product-listing-page/products.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
+:root {
+    --body-bg-color: rgb(243, 243, 243);
+    --page-bg-color: white;
```
Comment (akash-1318) [inline Product-listing-page/products.css:3]: use only one , hex color code or rgb

**Review by akash-1318 — COMMENTED** [review]
Just one suggestion mentioned have a look in that else looks good for me .

**Pull request merged**